components/index.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: single
+author_profile: false
+title: Zarr Components
+sidebar:
+  title: "Components"
+  nav: sidebar
+---
+
+Zarr consists of several components, both abstract and concrete. 
+These span both the physical storage layer and the conceptual structural layer. 
+Zarr-related projects all use the Zarr Protocol (and hence data model), described by the [Zarr Specification](https://zarr-specs.readthedocs.io/), but otherwise may choose to implement other layers however they wish.
+
+## Abstract components
+
+These abstract components together describe what type of data can be stored in zarr, and how to store it, without assuming you are working in a particular programming language, or with a particular storage system.
+
+**Protocol**: All zarr-related projects use the Zarr Protocol, described in the [Zarr Specification](https://zarr-specs.readthedocs.io/), which allows transfer of chunked array data and metadata between devices (or between memory regions of the same device). 
+The protocol works by serializing and de-serializing array data as byte streams and storing both this data and accompanying metadata via an [Abstract Key-Value Store Interface](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html#abstract-store-interface). 
+A system of [Codecs](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html#chunk-encoding) is used to describe the encoding and serialization steps.
+
+**Data Model**: The specification's description of the [Stored Representation](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html#stored-representation) implies a particular data model, based on the [HDF Abstract Data Model](https://support.hdfgroup.org/documentation/hdf5/latest/_h5_d_m__u_g.html). 
```

Comment:
i feel like it makes more sense to lead with the data model. the spec, i.e. the protocol, defines operations (create group, create array, write chunks to an array, etc) that only make sense in light of that particular data model.

Reply:
> the spec, i.e. the protocol

I think I disagree that these are one and the same (see https://github.com/zarr-developers/zarr-developers.github.io/pull/131#discussion_r2029606225), but otherwise agree with your point here.

Reply:
what's the difference between the contents of the zarr v2 / v3 specs and the zarr v2 / v3 protocols?

Reply:
See my long comment below: https://github.com/zarr-developers/zarr-developers.github.io/pull/131#issuecomment-2781516383